packages/dotcom-ui-shell/src/components/Shell.tsx

```diff
@@ -37,17 +38,14 @@ function Shell(props: TShellProps) {
     ...props.scripts,
     ...props.resourceHints,
     // TODO: abstract font URLs into 'core branding' package
```

Comment:
This comment can go

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/769/commits/f7cd27e948aadfb06b03d912c54cb80f62db4f7e).